gradle.properties

```diff
@@ -2,12 +2,13 @@ kotlin.code.style=official
 org.gradle.jvmargs=-Xmx4g -XX:+UseGCOverheadLimit -XX:GCTimeLimit=10 -Dfile.encoding=UTF-8
 org.gradle.parallel=true
 org.gradle.caching=true
+# TODO AGP blocker, fixed in 4.2.+
+# Plugin 'com.android.internal.library': registration of listener on 'TaskExecutionGraph.addTaskExecutionListener' is unsupported
+org.gradle.unsafe.configuration-cache=false
 # mandatory from AGP 3.6
 android.useAndroidX=true
 # Current stable version to be used in this project
 infraVersion=2021.1.7
-systemProp.kotlinVersion=1.4.21
```

Comment:
didn't see much profit in reading from system properties, othrwise it could be achieved with 

https://docs.gradle.org/current/userguide/configuration_cache.html#config_cache:requirements:undeclared_sys_prop_read

Reply:
https://github.com/avito-tech/avito-android/pull/711#discussion_r558271116